hack/test/aws-setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -eou pipefail
+
+REGION="us-east-1"
+BUCKET="talos-e2e"
+TMP=/tmp/e2e/aws
+
+## Setup svc account
+mkdir -p ${TMP}
+echo ${AWS_SVC_ACCT} | base64 -d > ${TMP}/svc-acct.ini
+
+# Ensure AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY env vars are available
+export AWS_ACCESS_KEY_ID=$(awk '/aws_access_key_id/ { print $NF }' ${TMP}/svc-acct.ini}
```

Comment:
We should either go with credentials file or use the env vars as part of the step. the AWS CLI will look for `~/.aws/credentials` so we could either drop everything we need in there, or we can skip that entirely and just use env vars.